### PR TITLE
Translate zh_cn/2014/10/27/changing-default-settings-of-ext-openssl

### PR DESCRIPTION
**Pull request by boyishwei**
Related #1074 (zh_cn)

98006631-5516-48e6-9596-0509b1d60994

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
@pmq20 @hlcfan @liuzxc @tzwm @happybai @diguage @alex-marmot @itsliupeng
@cassiuschen @1ternal @killernova @VincentZhao @reyesyang @jun1st @springwq
@Huibean @junzh0u @zigzagpig @42thcoder @GaaraLi @shouxian @johnnyhappy365
@aidewoode @Yunfei1982 @Airmacho @cursorzz @isjia @debbbbie @AndyHelix
@Zane5 @VirgilMing @boyishwei @dandananddada @jundechen @mohongshu

Please help review this, thanks.

File: zh_cn/news/_posts/2014-10-27-changing-default-settings-of-ext-openssl.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+---
+layout: news_post
+title: "更改 ext/openssl 默认设置"
+author: "usa"
+translator: "Ryan Jiang"
+date: 2014-10-27 12:00:00 +0000
+tags: 安全
```
Comment (stomar) [inline zh_cn/news/_posts/2014-10-27-changing-default-settings-of-ext-openssl.md]: `security` tag must **not** be translated!

**Comment by VincentZhao** [comment]
Missing some spaces between Chinese and English words. Please check.

File: zh_cn/news/_posts/2014-10-27-changing-default-settings-of-ext-openssl.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+1.9.3-p550
+: [r48121](http://svn.ruby-lang.org/cgi-bin/viewvc.cgi?revision=48121&view=revision)
+
+注意，回退这些更改将无法保证你的网络安全。
+在你回退之前你应该了解此举可能带来的影响。
+
+### 涉及到的 Ruby 包 
+
+这些更改体现在 net/http, net/imap 和 net/pop模块中。
```
Comment (hlcfan) [inline zh_cn/news/_posts/2014-10-27-changing-default-settings-of-ext-openssl.md]: net/pop 模块

File: zh_cn/news/_posts/2014-10-27-changing-default-settings-of-ext-openssl.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+具体如以下代码:
+
+{% highlight ruby %}
+ctx = OpenSSL::SSL::SSLContext.new
+ctx.set_params  # if you want to change some options, such as cert store, verify mode and so on, you can pass such parameters within a hash
+ssl = OpenSSL::SSL::SSLSocket.new(socket, ctx)
+{% endhighlight %}
+
+如何在客户端使用 ext/openssl， 我们认为此次变更不会带来任何影响。
```
Comment (hlcfan) [inline zh_cn/news/_posts/2014-10-27-changing-default-settings-of-ext-openssl.md]: Remove the extra space here.

**Comment by JuanitoFatas** [comment]
@hlcfan @VincentZhao @stomar Thanks for review, I fixed these minor issues and merged as c0ae887.

**Pull request closed without merging**